.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+
+defaults: &defaults
+  docker:
+    - image: circleci/node:6.12.3-browsers
+  working_directory: ~/axe-core
```

Comment:
Should this be `~/axe-core` @stephenmathieson?

Reply:
Nope, good catch. Copypasta fail.